app/src/main/java/com/octavianguzu/workshopandroid/model/Repository.java

```diff
@@ -0,0 +1,133 @@
+package com.octavianguzu.workshopandroid.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by octavian.guzu on 4/4/2017.
+ */
+
+public class Repository {
+    static final List<Repository> sMockRepository;
+
+    static {
+        sMockRepository = new ArrayList<>();
+
+        for (int i = 0; i < 100; i++) {
+            sMockRepository.add(new Repository(
+                    "Repo " + i,
+                    "Octocat",
+                    "Some repo",
+                    false,
+                    "master",
+                    new ArrayList<String>() {{
+                        add("Android");
+                        add("iOS");
+                    }},
+                    i,
+                    new Date(),
+                    new Date()
+            ));
+        }
+    }
+
+    String mName;
+    String mOwner;
+    String mDescription;
+    boolean mIsPrivate;
+    String mDefaultBranch;
+    List<String> mTopics;
+    int mWatchersCount;
+    Date mCreatedAt;
+    Date mUpdatedAt;
+
+    public Repository(String mName, String mOwner, String mDescription, boolean mIsPrivate, String mDefaultBranch, List<String> mTopics,
```

Comment:
Poti specifica in Android Studio (Preferences -> Code Style -> Java):
Fields sa aiba prefixul "m" si static fields sa aiba prefixul "s" -> Apply

Astfel cand iti va genera constructor + getters/setters sa tina cont de prefix si sa nu ajungi cu prefixul si in numele argumentelor sau al getterilor/setterilor

vezi urmatorul link: https://i.stack.imgur.com/g55ba.png